apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -158,6 +153,7 @@ const ConfigScreen = () => {
     renderPostAuthComponents &&
     !errors.projectSelection.projectNotFound &&
     !errors.projectSelection.cannotFetchProjects &&
+    !errors.projectSelection.protectionBypassIsDisabled &&
```

Comment:
I think it might be fine to follow the pattern here that we still render the next section even if this error exists, because it is simply a setting they have to change. Just like we allow them to continue to select the next section if the `projectSelection.invalidSpaceId` is true (which is a setting they need to change). Or perhaps we choose one line of logic or the other! wdyt? See video below for more context (on the weird standard we have for one but not the other). We kind of already have a pattern where things will NOT work with certain configurations but we still allow them to save most of the changes (i.e in the content type/preview path section we allow them to save empty fields even though that clearly won't work). 

https://github.com/contentful/apps/assets/58186851/76661919-44ac-4a3c-9228-2534e296f62f

Reply:
Sounds good with me!

Reply:
perhaps we remove this line from the conditional then?